wallet/src/ui/app/ApiProvider.ts

```diff
@@ -54,6 +55,14 @@ export default class ApiProvider {
         this._apiProvider = new JsonRpcProvider(DEFAULT_API_ENDPOINT);
     }
 
+    public setNewJsonRpcProvider(apiEnv: string) {
+        if (apiEnv) {
```

Comment:
I think if is unnecessary if we use `API_ENV`

Reply:
You are right. I take it out.